bundle/CMakeLists.txt

```diff
@@ -64,10 +64,17 @@ yoda_add_target_clean_all(
   "${CMAKE_BINARY_DIR}/protobuf-prefix"
 )
 
+if(NOT DEFINED PYTHON_EXECUTABLE)
+  find_package(PythonInterp 3.4 REQUIRED)
+endif()
+
 include("thirdparty/DawnAddProtobuf")
+
+install(DIRECTORY ${PROTOBUF_PYTHON_INSTALL}/google DESTINATION ${CMAKE_INSTALL_PREFIX}/python)
+
 include("thirdparty/DawnAddBoost")
 
-set(dawn_cmake_args -DProtobuf_DIR=${Protobuf_DIR} -DBOOST_ROOT=${BOOST_ROOT})
+set(dawn_cmake_args -DProtobuf_DIR=${Protobuf_DIR} -DBOOST_ROOT=${BOOST_ROOT} -DDAWN_PYTHON=ON -DDAWN_EXAMPLES=ON)

```

Comment:
I think that we should not build the examples by default. My suggestion is to have `make && make install` provide the default use-case with minimal building time. I know that the GT-team does this differently but their examples are basically codgen-tests. 

It is in my eyes debatable if we want to build DAWN_PYTHON by default (I would lean towards no). Happy to hear what people think. @Stagno do you have an opinion on this?

Reply:
here we are using these examples as integration tests. If we dont build it and test it we are delivering a functionality that is always broken

Reply:
or we could set the default in the options to OFF, and in the script of jenkins we activate it. 
Let me give it a try if you are happy with this solution

Reply:
That sounds like the best approach to me. Building is still on the "minimal path" and testing is more extended